example_config.yaml

```diff
@@ -7,6 +7,7 @@ cors_allowed_origins: null
 docs_url: /docs
 host: 127.0.0.1
 log_level: info
+mapping_url: /data/mapping.json
```

Comment:
I don't think this is needed.

Reply:
After a brief discussion, the approach taken makes sense. Especially with recent decisions of storing mappings.json outside of this repo.